android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -358,16 +358,22 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 return true;
 
             case R.id.action_share:
+                StringBuilder speakersNameBuilder  = new StringBuilder();
+                for(Speaker speaker : session.getSpeakers()){
+                    speakersNameBuilder.append(speaker.getName() + ", ");
+                }
+                speakersNameBuilder.delete(speakersNameBuilder.length() - 2, speakersNameBuilder.length() - 1);
```

Comment:
I believe you're deleting the comma and the space after the last name of the speaker.
Please add a comment above this line ```deleting comma and space after last name``` or you can use a clearer way by using a boolean flag.

Reply:
@srv-twry I added the comment 😅